bench_utils/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
```

Comment:
Not sure if it's worth improving this code further. However, `cargo clippy` gives many suggestions.

Reply:
Agree, there is currently the cnr_mkbench and mkbench which has a lot of code duplication, hopefully I can merge these again once I add the liveness changes to CNR, then it will be easier/faster to fix all the clippy issues as we only have to do it once